code/modules/mod/modules/_module.dm

```diff
@@ -418,7 +418,7 @@
 /obj/item/mod/module/anomaly_locked
 	name = "MOD anomaly locked module"
 	desc = "A form of a module, locked behind an anomalous core to function."
-	incompatible_modules = list(/obj/item/mod/module/anomaly_locked)
+	incompatible_modules = list()
```

Comment:
they still need to block themselves from being put in, otherwise you can put in several teleport modules

Reply:
i thought that was automatic

Reply:
impossible because subtypes would add themselves instead of their parent